imageio_ffmpeg/_utils.py

```diff
@@ -67,6 +67,16 @@ def _popen_kwargs(prevent_sigint=False):
             creationflags = 0x00000200
         else:
             preexec_fn = os.setpgrp  # the _pre_exec does not seem to work
+
+    if os.getenv("IMAGEIO_FFMPEG_NO_PREVENT_SIGINT", "").lower() not in (
+        "",
```

Comment:
black makes lists pointlessly stretched out :)

Reply:
Yeah Black is a blessing ... and a curse sometimes :)  -  I made a suggestion to clean it up a bit.